api/src/main/scala/Geoprocessing.scala

```diff
@@ -0,0 +1,65 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import java.util.concurrent.atomic.LongAdder
+
+import collection.concurrent.TrieMap
+
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+
+import geotrellis.raster._
+import geotrellis.raster.rasterize._
+import geotrellis.vector._
+import geotrellis.vector.io._
+
+import geotrellis.spark._
+
+trait Geoprocessing extends Utils {
+  /**
+    * For an InputJson object, return a histogram of raster grouped count results.
+    *
+    * @param   input  The InputData
+    * @return         A histogram of results in JSON format
+    */
+  def getRasterGroupedCount(input: InputData) = {
+    val aoi = createAOIFromInput(input)
+    val rasterLayers = createRasterLayerIds(input.rasters, input.zoom, aoi)
+    Map("result" -> rasterGroupedCount(rasterLayers, aoi)).toJson
```

Comment:
Sounds good. I think the next op is `Map[String, Double]` but we can handle figuring out a `Numeric[T]` constraint or something in #65, so I'll leave it as `[String, Int]` for now.